Configuration_adv.hpp

```diff
@@ -460,7 +462,9 @@
   #define SUPPORT_INFO_DISPLAY         1
 #else
   #define SUPPORT_POINTS_OF_INTEREST 0
-  #define SUPPORT_GUIDED_STARTUP     0
+  #ifndef SUPPORT_GUIDED_STARTUP       
+    #define SUPPORT_GUIDED_STARTUP   0
+  #endif
```

Comment:
If there is no display available, this change would allow the user to define `SUPPORT_GUIDED_STARTUP=1`. we should rather check if it is set, and if it is 1, throw an error

Reply:
Ahh, good point.